tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors_test.ts

```diff
@@ -104,6 +207,84 @@ describe('debugger selectors', () => {
     });
   });
 
+  describe('getAlertTypesOfVisibleExecutionDigests', () => {
+    it('returns all-null array when there is no focused alert type', () => {
+      const state = createState(
+        createDebuggerState({
+          activeRunId: '__default_debugger_run__',
```

Comment:
Btw, general comment. If this ID is not useful for testing the method, consider omit it and just populate one in the fixture factory method. I try to only override the parts that is supposed to be relevant to the test.

Reply:
Ack. In this particular case, the `activeRunId` is not needed. Removed it.